webapp/src/Controller/API/ProblemController.php

```diff
@@ -332,8 +332,8 @@ public function linkProblemAction(
 
         $contest = $this->em->getRepository(Contest::class)->find($this->getContestId($request));
 
-        $lazyEvalResults = null;
-        if ($contestProblemPut->lazyEvalResults !== null) {
+        $lazyEvalResults = 0;
+        if ($contestProblemPut->lazyEvalResults !== DOMJudgeService::LAZY_DEFAULT) {
             $lazyEvalResults = (int)$contestProblemPut->lazyEvalResults;
         }
```

Comment:
Why not:
```suggestion
$lazyEvalResults = ($contestProblemPut->lazyEvalResults;
```

It's always an int now with a default of 0.

Reply:
If only I actually committed what I discussed.